### PR TITLE
[enterprise-4.14] CNV#33789: RN - Cannot SSH into VM over OVNKubernetes

### PR DESCRIPTION
**Pull request by danielclowers**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/CNV-33789
https://issues.redhat.com/browse/CNV-24889

Link to docs preview:
https://66905--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -224,6 +224,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2055595*])
 ** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
+//CNV-33789: Cannot SSH into VM over NodePort and Console's FQDN when using OVNKubernetes networking
+* You cannot SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2165895)[*BZ#2165895*])
```
Comment (jeana-redhat) [inline virt/release_notes/virt-4-14-release-notes.adoc:228]:
```suggestion
* You cannot SSH into a VM when using the `networkType: OVNKubernetes` option in your `install-config.yaml` file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2165895[*BZ#2165895*])
```

**Review by jeana-redhat — COMMENTED** [review]
Link typo.

Here's the preview link for future review: https://66905--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-known-issues

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: virt/release_notes/virt-4-14-release-notes.adoc (hunk 1 above)
Comment (danielclowers) [reply virt/release_notes/virt-4-14-release-notes.adoc]: thanks for spotting that! and sorry i forgot the preview link. edited the OP to include it :)

Comment (jeana-redhat) [reply virt/release_notes/virt-4-14-release-notes.adoc]: no worries! lots going on this week :)

**Pull request closed without merging**